mllm-kernel/mllm_kernel/cpu/jit/add_constant.py

```
@@ -0,0 +1,122 @@
+
+    Example:
+        >>> import torch
+        >>> from mllm_kernel.cpu.jit import add_constant
+        >>> x = torch.randn(1024)
+        >>> y = add_constant(x, 16)  # y = x + 16
+    """
+    if constant not in (1, 2, 4, 8, 16):
+        raise ValueError(
+            f"Constant must be one of [1, 2, 4, 8, 16], got {constant}. "
+            "Use add_constant_runtime for arbitrary constants."
+        )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)